src/main/java/duke/manager/TaskManager.java

```diff
@@ -0,0 +1,197 @@
+package duke.manager;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.Scanner;
+
+public class TaskManager {
+
+    /**
+     * Prints a line on the console
+     */
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Determines the task type,
+     * assuming it is the first
+     * word in the message and
+     * returns it
+     *
+     * @param message The input by the user.
+     * @return The first word of the string.
+     */
+    private static String taskType(String message) {
+        String[] type = message.split(" ");
```

Comment:
I think the split method called for the message variable does not have a catch exception solution here. Adding an error catch statement would make the software better in performance, as the message variable contains user input (which can be almost anything).